topic_6/src/main/java/com/globant/user/controller/UserRepositoryController.java

```diff
@@ -0,0 +1,49 @@
+package com.globant.user.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.globant.user.User;
+import com.globant.user.UserRepository;
+
+/**
+ * This controller handle request input and generate response output of users repository
+ * @author Herminio Andres Hernandez
+ * @version 01/08/2018
+ */
+@RestController
+@RequestMapping("/registration")
+public class UserRepositoryController {
+
+	UserRepository userRepository = new UserRepository();
+
+	@RequestMapping("/createUser")
```

Comment:
Too much GET and no other methods, REST core principle is that HTTP methods represent the different possible operations.
You used them in the swagger files but not in the code, you can do something like
@RequestMapping(value = "/ex/foos", method = POST)

or

use the annotations like

GetMapping, PutMapping, DeleteMapping, PatchMapping, RequestMapping